extension/src/validator/validator-builder.js

```diff
@@ -146,4 +129,17 @@ function withPayment (paymentObject) {
   }
 }
 
+function invalidJSON (requestString) {
```

Comment:
It's quite hard to read negative function. Better is to have `isValidJson()`.

Reply:
but then you will check it with an `!`

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/232/commits/19456341b3d610e4999a990dafc5cc7f19f7a548